app/admin/plugins.rb

```diff
@@ -1,18 +1,85 @@
 ActiveAdmin.register Plugin do
 
-  # See permitted parameters documentation:
-  # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
-  #
-  # Uncomment all parameters which should be permitted for assignment
-  #
-  # permit_params :plugin_name, :url_id, :status, :type, :version, :other_data
-  #
-  # or
-  #
-  # permit_params do
-  #   permitted = [:plugin_name, :url_id, :status, :type, :version, :other_data]
-  #   permitted << :other if params[:action] == 'create' && current_user.admin?
-  #   permitted
-  # end
-  
+  actions :index, :show  
+  filter :plugin_name
+  filter :url_id
+  filter :status , :as => :select, :collection => Plugin::STATUS.invert
+  filter :first_seen
+  filter :last_seen
+  filter :created_at
+  filter :updated_at
+
+  index do 
+    id_column
+    column 'Name' do|plugin|
+      slug_name = PluginSlug.where("slug LIKE?", "#{ plugin.plugin_name }").first
```

Comment:
why are you not storing the associated plugin slug to the plugin table as soon as you're fetching it in the scraping job, are you going to query it each time?

Reply:
query it once and store it directly while you're running the scraping job